styles.css

```diff
@@ -494,5 +561,369 @@ textarea {
+  #welcome-text {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  .my-socials {
+    order: -1;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 4rem;
+    gap: 13px;
+  }
+
+  #work-section {
+    display: grid;
+    justify-content: center;
+    gap: 24px;
+    max-width: 60%;
+    margin: 0 auto;
+  }
+
+  .big-image {
+    background-image: url('images/image2.png');
+    background-repeat: no-repeat;
+    height: 448px;
+  }
+
+  .recent-work {
+    /* max-width:60%; */
+  }
+
+  .title-recent-word {
+    display: grid;
+    grid-template-columns: 0.45fr 1fr;
+    margin-top: 100px;
+  }
+
+  #recent-work-title {
+    text-align: left;
+    align-self: center;
+  }
+
+  .title-recent-word hr {
+    align-self: center;
+  }
+
+  .multipost {
+    display: grid;
+    grid-template-columns: 1fr 0.55fr;
+    margin-top: 50px;
+  }
+
+  #project-description-title {
+    margin-top: -3px;
+  }
+
+  #project-description {
+    color: #091e42;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  #img-project {
+    display: none;
+  }
+
+  #card-section {
+    grid-template-columns: 1fr 1fr 1fr;
+    column-gap: 20px;
+    row-gap: 37px;
+    padding-bottom: 130px;
+  }
+
+  .post:nth-child(1) {
+    background-image: url('images/Mask\ Group.png');
+  }
+
+  .post:nth-child(2) {
+    background-image: url('images/card2.png');
+  }
+
+  .post:nth-child(3) {
+    background-image: url('images/card3.png');
+  }
+
+  .post:nth-child(5) {
+    background-image: url('images/card2.png');
+  }
+
+  .post:nth-child(6) {
+    background-image: url('images/card3.png');
+  }
+
+  .post1 .card-contents {
+    display: none;
+  }
+
+  .card-big-btn {
+    display: none;
+    z-index: 1;
+  }
+
+  .post1 .card-big-btn {
+    margin-top: 340px;
+  }
+
+  #cursor {
+    position: absolute;
+    height: 16px;
+    width: 14px;
+    margin-top: 200px;
+    margin-left: 80px;
+  }
+
+  .card-main-title1 {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #fff;
+    width: 20.563rem;
+  }
+
+  h3.card-main-title {
+    display: none;
+  }
+
+  .post2 .card-contents {
+    margin-top: 20px;
+  }
+
+  .post4 .card-contents {
+    margin-top: 20px;
+  }
+
+  .post5 .card-contents {
+    margin-top: 20px;
+  }
+
+  .post3 .card-contents {
+    margin-top: 62px;
+  }
+
+  .post6 .card-contents {
+    margin-top: 62px;
+  }
+
+  .post:hover .card-big-btn {
+    display: block;
+    cursor: pointer;
+  }
+
+  .About-me {
+    padding: 4% 0 4% 0;
+    background-image: url('images/illustration-about\ me@2x.png');
+    background-size: 28rem;
+    background-repeat: no-repeat;
+    background-position: right;
+  }
```

Comment:
As mentioned by the previous reviewer:
> * [ ]  I think the `about me` section also needs an improvement. 🪐   On screen sizes between 900px and 1230px your background image in this section contrasts with the text on your website, This makes it difficult to read the text. Please consider fixing this. -: U can try reducing the background image size at this screen size 👍
> * Screenshot:
>   ![Screenshot 2023-04-01 094635](https://user-images.githubusercontent.com/107632780/229277276-128e271f-3ada-4492-b43a-d42500b10bd0.png)

- [ ] Please make sure the background image doesn't overflow with the text. You can do that by using % intead of rem (updating `background-size: 28rem;` to `background-size: 35%`). 
This will make your project way more prettier which will improve the user experience 📱  😄 👍